17/windows/windowsservercore-ltsc2019/Dockerfile

```diff
@@ -24,18 +24,8 @@
 ARG version=3107.v665000b_51092-5
 FROM jenkins/agent:${version}-jdk17-windowsservercore-ltsc2019
 
+ARG version=3107.v665000b_51092-5
```

Comment:
looks like a mistake
```suggestion
```

Reply:
It is not. I'll add a comment as your are the 2nd reviewer to stumble on this one.

The explanation is in the footer of the PR body:

> There was an issue with the ARG version in the 11/windows/windowsservercore-ltsc2019/Dockerfile and 17/windows/windowsservercore-ltsc2019/Dockerfile files (ref. https://docs.docker.com/engine/reference/builder/#scope) fixed in this PR

The scope of the `ARG` instruction on line 24 is closed by the `FROM` instruction on line 25, making the variable `$version` empty when using it in the instructions. That is a constraint from the docker builder system.

Please note that:
- The updatecli process is set up to update all occurences of the `ARG` so not a problem here
- Calling `docker build --build-arg version=xxxx` is expected to replace ALL `ARG` instructions, so it's also ✅